api/src/main/java/data/Drills/Answer.java

```diff
@@ -0,0 +1,32 @@
+package data.Drills;
+
+/**
+ * Created by Michaela Bamburová on 13.05.2016.
+ */
+public class Answer {
+
+	private final boolean isCorrect;
+	private final String text;
+
+	public Answer(boolean isCorrect, String text) {
+		this.isCorrect = isCorrect;
```

Comment:
přemýšlím, že tady by to taky chtělo idčko, ale vzhledem k tomu že to bude na mongo, tak to asi nebude zase tolik potřeba, protože tohle bude jako embed v Question